nsv13/code/modules/research/techweb/all_nsv_nodes.dm

```diff
@@ -261,3 +261,15 @@
 	design_ids = list("borg_upgrade_muniapp")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
 	export_price = 5000
+
+/// Munitions Upgrade for Energy Weapons
+/datum/techweb_node/Weapon_Cooling
+	id = "Weapon_Cooling"
```

Comment:
```suggestion
/datum/techweb_node/weapon_cooling
	id = "weapon_cooling"
```
Please try to avoid capitalizing typepaths, and also id names in this case.